packages/marko-web-html-sitemap/routes/index.js

```diff
@@ -0,0 +1,150 @@
+
+const moment = require('moment');
+const gql = require('graphql-tag');
+const { asyncRoute } = require('@parameter1/base-cms-utils');
+const dateListTemplate = require('../templates/date-list');
+const dayTemplate = require('../templates/day');
+
+const FORMAT = 'YYYY-MM-DD';
+const dateNotFound = () => {
+  const error = new Error('The requested date was not found.');
+  error.statusCode = 404;
+  return error;
+};
+const invalidDate = () => {
+  const error = new Error('The requested date is invalid.');
+  error.statusCode = 400;
+  return error;
+};
+
+const ALL_PUBLISHED_CONTENT_DATES = gql`
+  query AllPublishedContentDates($input: AllPublishedContentDatesQueryInput = {}) {
+    allPublishedContentDates(input: $input) {
+      id,
+      year,
+      month,
+      day,
+      count
+    }
+  }
+`;
+
+const ROOT_ALIAS = '/html-sitemap';
+
+module.exports = (app) => {
+  app.get(`${ROOT_ALIAS}/:year(\\d{4})/:month(\\d{2})/:day(\\d{2})`, asyncRoute(async (req, res) => {
```

Comment:
Allow this to be passed in from the site:

```suggestion
module.exports = ({
  mountPoint: 'sitemap',
}) => (app) => {
  app.get(`${mountPoint}/:year(\\d{4})/:month(\\d{2})/:day(\\d{2})`, asyncRoute(async (req, res) => {
```

And in the consuming application:
```js
const router = require('@parameter1/base-cms-marko-web-html-sitemap/routes');

module.exports = (app) => {
  app.use(router({ mountPoint: 'some-wacky-sitemap-prefix' }));
};
```

Reply:
@solocommand  I couldn't get that version to resolve, but i did the following:

in the routes/index.js 
`module.exports = (app, { mountPoint } = { mountPoint: '/html-sitemap' }) => {`

if you do the following on use of that route
`htmlSitemap(app);`
The mountPoint will be  /html-sitemap
if you do
`htmlSitemap(app, { mountPoint: '/wacky-site' });`
The mountPoint will be /wacky-site instead.